cranelift-codegen/src/verifier/liveness.rs

```diff
@@ -106,7 +106,9 @@ impl<'a> LivenessVerifier<'a> {
                         Some(lr) => lr,
                         None => return fatal!(errors, inst, "{} has no live range", val),
                     };
-                    if !self.live_at_use(lr, inst) {
+
+                    let ebb = lr_ctx.order.inst_ebb(inst).unwrap();
```

Comment:
Should this be `let ebb = lr_ctx.order.inst_ebb(inst).ok_or(...)?;`?

Reply:
If I'm reading this right, either the new `ebb` should be the same value as the previous binding (in which case why compute it) or we should assert that they are the same value?

Reply:
Indeed, we can assert it was the same. Previous checks verify that this instruction has been inserted, so it's fine to unwrap here.